inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java

```diff
@@ -133,19 +135,23 @@ private List<InsertData> getInsertServiceList() {
 
     private List<MQInfo> getClusterFromManager() {
         Properties properties = new Properties();
+        List<MQInfo> mqConfig;
         try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream(DEFAULT_CONFIG_PROPERTIES)) {
             properties.load(inputStream);
             String managerHosts = properties.getProperty("manager.hosts");
             String clusterTag = properties.getProperty("proxy.cluster.tag");
             String[] hostList = StringUtils.split(managerHosts, ",");
             for (String host : hostList) {
-                List<MQInfo> mqConfig = getMQConfig(host, clusterTag);
-                if (ObjectUtils.isNotEmpty(mqConfig)) {
-                    LOG.info("return mqConfig");
-                    return mqConfig;
+                while (true) {
```

Comment:
Do we need to set a maximum number of retries? If it exceeds, it will fail, otherwise, the program will loop infinitely.

Reply:
there is no need to set a maximum number of retries, because MQ is a necessary service for audit and other InLong components, we don't know when the users would configure the MQ cluster, it's better to keep waiting.

Reply:
Ok, let's change it like this and keep watching.